edison.coffee

```diff
@@ -84,6 +84,54 @@ module.exports =
 			path: '/config.json'
 
 	initialization:
+		dependencies:
+			osx: [
+					name: 'greadlink'
+					instructions: '''
+						This utility is included in the `coreutils` package. Run:
+
+							$ sudo port install coreutils
+
+						Or with Homebrew:
+
+							$ brew install coreutils
+					'''
+				,
+					name: 'dfu-util'
+					instructions: '''
+						Install dfu-util by running:
+
+							$ sudo port install dfu-util@0.7
```

Comment:
@shaunmulligan Should I remove the `port install` one and keep the `brew` one? I copied this from the current instructions.